src/pages/using-spark/foundations/color-usage.mdx

```diff
@@ -123,15 +122,15 @@ Following the August 2020 update to the Rocket Orbit color palette, it’s impor
 <SprkDivider element="span" additionalClasses="sprk-u-mvn"/>
 <h2 class="docs-b-h2 sprk-b-TypeDisplayTwo sprk-u-mbl sprk-u-Measure">Our Color Story</h2>
 <p>
-  We designed the Orbit palette to be modern and approachable, while fitting our brand character and highlighting color psychology attributes.
+  The Orbit palette is inspired by the colors seen from a rocket as it enters the edge of space.
```

Comment:
Before: 
![image](https://user-images.githubusercontent.com/4342363/94458207-a37aa700-0183-11eb-8bca-5a861315fe49.png)


After: ![image](https://user-images.githubusercontent.com/4342363/94458172-92319a80-0183-11eb-9f32-c034afccb629.png)

Reply:
so we're considering this change because we believe quote doesn't actually add that much story to our colors.

This quote is about a team member **being reminded** of a moment in history. It's not actually the inspiration for the color palette. 

We're also inadvertently connecting our brand to Felix Baumgartner, and Redbull. Which doesn't feel apt

Reply:
Those are good points. I would just make sure to run it by Jane and Jason to make sure we aren't losing somehting they really care about that could maybe be rewritten to keep the spirit of the story without the specifics

Reply:
Great point! Gonna pitch something like this to them

![image](https://user-images.githubusercontent.com/4342363/94579931-5102be80-0247-11eb-8579-3709b9611ce6.png)

Reply:
Jane, Brielle and Jason liked our changes!